README.md

```diff
@@ -41,3 +41,8 @@ We have support for a number of CI systems for a few reasons:
 # Releases
 SharpPcap is released via nuget
 
+# Future release plans
+- 5.4 is likely to be the last release of the 5.x series
+- 6.0 will contain a number of API breaking changes including:
+  - Removal of deprecated WinPcap class
```

Comment:
If you are going to make a breaking change, you may as well remove the Npcap classes since all their features would be covered by Libpcap classes